### DISCUSSION TITLE
Renal Impairment population

### DISCUSSION THREAD

**Discussion by noxavier** [Open-Systems-Pharmacology/Forum#1480] [Questions & Problems] [answered]
According to the OSPS tutorial and the article by [Edgington et al. 2020](https://accp1.onlinelibrary.wiley.com/doi/10.1002/jcph.1713), I am trying to construct a population with Chronic Kidney Disease. However, the parameters calculated by the equations in the article are not the same as those in the OSPS tutorial attachment. And there is no clear calculation method in the OSPS tutorial.

So how do I set the parameters for the CKD population?

![image](https://user-images.githubusercontent.com/109507330/225864056-1d71e9b4-84a5-4f3b-9731-273fb6ca66c3.png)
![image](https://user-images.githubusercontent.com/109507330/225864098-7df789c0-0c4d-4187-83d9-34a75139ffc9.png)
![image](https://user-images.githubusercontent.com/109507330/225864179-7401571b-1637-41f4-9607-2fd0b4fab7ed.png)

**Comment by Yuri05** [comment, marked as answer]
PK-Sim provides creation of CKD individuals and populations out of the box now. The implemented algorithm is the one from the paper 
[_Malik, P.R.V., Yeung, C.H.T., Ismaeil, S., Advani, U., Djie, S. and Edginton, A.N. (2020), A Physiological Approach to Pharmacokinetics in Chronic Kidney Disease. The Journal of Clinical Pharmacology, 60: S52-S62. https://doi.org/10.1002/jcph.1713_](https://doi.org/10.1002/jcph.1713)

https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-creating-individuals#chronic-kidney-disease-ckd

- First, you need to create a CKD **Individual**

![grafik](https://user-images.githubusercontent.com/25061876/226943668-37562804-afbf-4f0e-ac33-3cbff9922612.png)

- In the next step, you need to create a **population** based on this CKD Individual

![grafik](https://user-images.githubusercontent.com/25061876/226944593-fbe64fc0-1d2c-412e-a23e-376494064d01.png)

**Reply by prvmalik** [reply]
Important to note that this considers foundational anatomy and physiology parameters only. Enzyme/transporter expression profile in CKD must still be inserted by the user, based on as much scientific literature can be obtained. If no data are available for a particular enzyme or transporter of interest in the literature, then a generic parameter from Sayama et al. 2014 [10.1208/s12248-014-9626-3](https://doi.org/10.1208%2Fs12248-014-9626-3) could be helpful. 

Available to answer any questions. Unclear what the original question is or what the OSPS tutorial is.

**Reply by HaiXxxxxx** [reply]
Dear @Yuri05,
I found that the built-in CKD module in PK-Sim were only for individuals with eGFR below 60mL/min/1.73 m2. What parameters should I change if I want to set up an individual with mild renal impairment, i.e. eGFR between 60-90mL/min/1.73 m2? 
Thanks.

**Reply by prvmalik** [reply]
There is not data to support that kidney volume, renal blood flow, gastrointestinal transit, plasma protein binding etc. are meaningfully compromised in mild renal impairment
